services/localazy/localazy-lang-progress.service.js

```diff
@@ -0,0 +1,86 @@
+'use strict'
+
+const Joi = require('joi')
+const LocalazyBase = require('./localazy-base.service')
+
+const queryParamSchema = Joi.object({
+  locale_display: Joi.string().allow('loc', 'en').optional(),
+})
+
+module.exports = class LocalazyLangProgress extends LocalazyBase {
+  static route = this.buildRoute(
+    'lang-progress',
+    ':localeCode',
+    queryParamSchema
+  )
+
+  static examples = [
+    {
+      title: 'Localazy language progress with locale',
+      keywords: this.keywords,
+      documentation: this.documentation,
+      namedParams: {
+        projectId: 'floating-apps',
+        localeCode: 'pt_BR',
+      },
+      staticPreview: this.render({
+        message: '100%',
+        label: 'pt_BR',
+        color: '#066fef',
+      }),
+    },
+    {
+      title: 'Localazy language progress in English',
+      keywords: this.keywords,
+      documentation: this.documentation,
+      namedParams: {
+        projectId: 'floating-apps',
+        localeCode: 'pt_BR',
+      },
+      staticPreview: this.render({
+        message: '100%',
+        label: 'Brazilian Portuguese',
+        color: '#066fef',
+      }),
+      queryParams: {
+        locale_display: 'en',
+      },
+    },
+    {
+      title: 'Localazy language progress with localized name',
+      keywords: this.keywords,
+      documentation: this.documentation,
+      namedParams: {
+        projectId: 'floating-apps',
+        localeCode: 'pt_BR',
+      },
+      staticPreview: this.render({
+        message: '100%',
+        label: 'Português (BR)',
```

Comment:
I like the concept of displaying the language name in the language itself, that's a nice user experience. 👍🏻